### PR TITLE
Update to make jekyll-haml work with jekyll ~> 4.0

### PR DESCRIPTION
**Pull request by mfaughn**

### CODE REVIEW

**Comment by mfaughn** [comment]
I created some test files for making sure that haml files, partials, and haml tag work.  Tests aren't automated.  Not sure if my files should be added but it would be good to have some sort of tests in this repo...

File: lib/jekyll-haml/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Jekyll
   module Haml
-    VERSION = '0.1.6'.freeze
+    VERSION = '2.0.0'.freeze
```
Comment (samvincent) [inline lib/jekyll-haml/version.rb:3]: This version doesn't match the changeling, but otherwise happy to merge.

**Review by samvincent — CHANGES_REQUESTED** [review]

File: lib/jekyll-haml/version.rb (hunk 1 above)
Comment (mfaughn) [reply lib/jekyll-haml/version.rb]: Oops.  Fixed that.  Should be OK now.

Comment (samvincent) [reply lib/jekyll-haml/version.rb]: Thank you. Appreciate the update.

Comment (samvincent) [reply lib/jekyll-haml/version.rb]: @mfaughn I forgot to push the new version until just now. Hope that works well for you.
https://rubygems.org/gems/jekyll-haml/versions/1.0.0

Comment (mfaughn) [reply lib/jekyll-haml/version.rb]: OK by me.  FWIW, the powers that be prevent me from using any Jekyll plugins at all anymore.  Hopefully this changes in the future...

**Pull request merged**